autoscale-workload-rabbit/pom.xml

```diff
@@ -52,6 +52,16 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.inject</groupId>
+            <artifactId>jersey-hk2</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.media</groupId>
+            <artifactId>jersey-media-json-jackson</artifactId>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
Why would the service have two extra runtime dependencies just to add some tests?

Reply:
These were missing from the previous PR which I found when I added the tests.

Reply:
So these are actually required for it to work and it is not actually fixed as things stand?

Reply:
Yes